src/index.html

```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
                                         <li class="accordion-item" data-accordion-item>
                                             <a href="#collapse18" class="accordion-title"><h5>What accessions are administered by the EVA?</h5></a>
                                             <div class="accordion-content" data-tab-content>
-                                                The EVA follows the SRA model for accessioning:
+                                                <p>
+                                                    The EVA follows the SRA model for accessioning:
                                                 <ul>
                                                     <ul>
-                                                    <li>A submitted project is administered a ‘project’ accession. These begin “PRJEB” followed by a numerical sequence.</li>
-                                                    <li>Each analysis object within a project is administered an ‘analysis’ accession. These begin “ERZ” followed by a numerical sequence.</li>
+                                                        <li>A submitted project is administered a ‘project’ accession. These begin “PRJEB” followed by a numerical sequence.</li>

```

Comment:
Please check the quotes used in this text. Maybe it is a result of copying them from Google Docs, but neither the single or double ones are not ASCII characters and could cause encoding problems, see:

`’` versus `'`
`“` versus `"`